src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -60,38 +61,60 @@ public void MefCtor_CheckIsExported()
         }
 
         [TestMethod]
-        public void SynchronizeWithServer_FailureToCheckServerConnection_CriticalException_ExceptionThrown()
+        public async Task SynchronizeWithServer_NonCriticalException_ExceptionCaughtAndLogged()
```

Comment:
it seems this test and the one below is mostly the same they only assert different things. Maybe they can be merged into one.